bin/travis_clang_format.sh

```diff
@@ -13,7 +13,7 @@ if [[ "${TEST_CLANG_FORMAT}" == "yes" ]]; then
         exit 1
     fi
 
-    FILES=`git ls-files | grep -E "\.(cpp|h|hpp|c)$" | grep -Ev "symengine/utilities" | grep -Ev "cmake/"`
+    FILES=`git ls-files | grep -E "\.(cpp|h|hpp|c)$" | grep -Ev "symengine/utilities" | grep -Ev "cmake/" | grep -Ev "symengine/parser/tokenizer.cpp"`
```

Comment:
Can you revert this change as `bin/test_format_local.sh` is updated to format this file as well?

Reply:
We can format that file.

Reply:
OK. I formatted the file and reverted this.